numpy/core/fromnumeric.py

```diff
@@ -2820,6 +2820,18 @@ def amax(a, axis=None, out=None, keepdims=np._NoValue, initial=np._NoValue,
     --------
     max : alias of this function
     ndarray.max : equivalent method
+
+    Examples
+    --------
+    >>> np.amax([4])
+    4
+    >>> np.amax([4,-1, 67, 89])
+    89
+    >>> np.amax([[1,2,3,46,44],[2,3,4,55,5]], axis=0)
```

Comment:
```suggestion
    >>> np.amax([[1, 2, 3, 46, 44], [2, 3, 4, 55, 5]], axis=0)
```
I prefer the example to follow pep8. (not sure this is required, but the example at https://numpydoc.readthedocs.io/en/latest/format.html#examples follows the style as well)